test/vstest/runtests.cmd

```diff
@@ -0,0 +1,7 @@
+set target=%1
+set version=%2
```

Comment:
I recommend we add another variable here since we frequently consume new runtimes before SDKs.

```suggestion
set sdk_version=%2
set runtime_version=%3
```

Then, add a second call to dotnet-install.ps1 to install the runtime:

    ....   dotnet-install.ps1 -Runtime dotnet -Version %runtime_version% ....